modules/vaos/app/services/vaos/configuration.rb

```diff
@@ -14,6 +14,10 @@ def service_name
       'VAOS'
     end
 
+    def key
```

Comment:
NITPICK: possibly consider calling this "cert" or "private_key" as "key" is kind of generic and could be confused with "hash key"

Reply:
Used to be called `cert_key` but changed it as I liked @omgitsbillryan suggestion to make it more succinct 😃Variable naming ~fight~ discussion... go! 🥊

Reply:
`rsa_key` makes sense to me